ts/src/pro/bingx.ts

```diff
@@ -224,6 +237,206 @@ export default class bingx extends bingxRest {
         }, market);
     }
 
+    async watchTickers (symbols: Strings = undefined, params = {}): Promise<Tickers> {
+        /**
+         * @method
+         * @name bingx#watchTickers
+         * @description watches a price ticker, a statistical calculation with the information calculated over the past 24 hours for all markets of a specific list
+         * @see https://bingx-api.github.io/docs/#/en-us/swapV2/socket/market.html#Subscribe%20to%2024-hour%20price%20changes%20of%20all%20trading%20pairs
+         * @param {string[]} symbols unified symbol of the market to watch the tickers for
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @returns {object} a [ticker structure]{@link https://docs.ccxt.com/#/?id=ticker-structure}
+         */
+        await this.loadMarkets ();
+        symbols = this.marketSymbols (symbols, undefined, true, true, false);
+        let firstMarket = undefined;
+        let marketType = undefined;
+        const symbolsDefined = (symbols !== undefined);
```

Comment:
the signature of the method does not allow undefined `symbols`, at most, they can be empty

Reply:
wdyt? we allow users `watchTickers (undefined, {type:'swap'})`

Reply:
![image](https://github.com/ccxt/ccxt/assets/43336371/01d88dc3-7747-4dea-a6b6-e4faff6688b9)
![image](https://github.com/ccxt/ccxt/assets/43336371/7bc92e9d-ebff-4370-b054-55ed6367e873)

there is a difference in the signature

Reply:
i have replie don this on discrod, to mention again, they are different methods, having different base signature, so that change should not be implied in this PR.

Reply:
I think you missed the point, because the signature is different doing symbols !== undefined does not make sense